src/main/java/bio/terra/service/search/SearchService.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.service.search;
+
+import bio.terra.common.exception.PdaoException;
+import bio.terra.model.SearchIndexModel;
+import bio.terra.model.SearchIndexRequest;
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.service.tabulardata.google.BigQueryPdao;
+import org.apache.http.HttpHost;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestClient;
+import org.elasticsearch.client.RestClientBuilder;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.client.indices.CreateIndexRequest;
+import org.elasticsearch.client.indices.CreateIndexResponse;
+import org.elasticsearch.common.settings.Settings;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class SearchService {
+
+    private final BigQueryPdao bigQueryPdao;
+    private final RestHighLevelClient client;
+
+    @Autowired
+    public SearchService(BigQueryPdao bigQueryPdao) {
+        // needs to be moved into own class with config
```

Comment:
Making the service with a config could've be done as a separate ticket, but may need to be done before this PR can be committed.

Reply:
Also, we may want to make a ticket to add ES to the `StatusService.getStatus()` response, if it is enabled.

Reply:
I created https://broadworkbench.atlassian.net/browse/DR-1831 for the getStatus() fix.